Contact form

Rophah opened the pull request:
This is the contact form section of the portfolio page.

Rophah committed:
feat: added work and about me section
fix

Rophah committed:
Delete test.css

Rophah committed:
fix: fixed all linters errors
fix: added margin to the anchor tag in the about section

Rophah committed:
Fix: Deleted test.css as  it is not needed

Rophah committed:
Merge branch 'mobile' of https://github.com/Rophah/Portfoliopage into mobile

Rophah committed:
comment new section

Rophah committed:
fix: removes the second <h2>
fix: changes the <p> in let's connect to <h5>
fix: changes the technology categories and subcategories to a list using <ul> and <li>
style: removes the font-weight: 500 on line 263 and creates a class of fontweight500 with property 'font-weight: 500' so as to make my code dry

Rophah committed:
fix: runs linter

Rophah committed:
fix: creates <ul> inside the <li> to line 283
fix: creates <ul> inside the <li> to line 302

Rophah committed:
feat: creates contact form
feat: adds all images on the contact form

Rophah committed:
fix: runs npm run format:check

Rophah committed:
feat: adds the hover property to get in
touch button

Rophah committed:
fix: runs all required linters

adejam commented on index.html:
```diff
@@ -84,5 +84,284 @@
+      <img src="images/contact/rectangle7.png" alt="rectangle7" class="rectangle7contact" />
+
+      <form action="https://formspree.io/f/xbjwbpyj" method="POST" class="contactform">
+        <input
+          class="formradius forminput"
+          type="text"
+          placeholder="Your name"
+          maxlength="30"
+          required
+        />
+        <input class="formradius forminput" type="email" placeholder="Email" required />
```
your `name` and `email` inputs has no name attribute hence are not submitted with your form as part of the payload  to the formspree api

adejam commented on index.html:
```diff
@@ -84,5 +84,284 @@
+          maxlength="500"
+          required
+        >
+        Write your message here</textarea
```
This is a bad practice, you should use place holder rather than this. 
This cause the "write your messages" to be seen as a value rather than a placeholder.
Also because you wanted to mimic it to look like a placeholder, you made the text faint which is a bad UI. 

Please use a placeholder attribute instead and correct the the text input value color

adejam commented on index.html:
```diff
@@ -84,5 +84,284 @@
+        >
+        Write your message here</textarea
+        >
+        <button type="submit" class="formbutton fontweight500">Get In Touch</button>
```
The button's pointer is using the default arrow pointer rather than the `cursor` pointer

adejam reviewed (changes requested):
# Still needs a couple of changes

Please let's work on the reviews and make changes accordingly.
Also please avoid having a lot of changes due to format issues, makes it hard to know what was changed in the file

Rophah committed:
fix: corrects and creates message placeeholder in
the textarea
feat: makes the cursor a pointer cursor
fix: gives name attribute to all input fields

adejam reviewed (approved):
# LGTM

Please just fix the textarea color and  merge. Goodluck on your next milestone
